### PR TITLE
Thumbnail grid: changes according to guideline

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] https://github.com/Samsung/TAU/issues/1354
[Problem] Thumbnail grid needs update
[Solution]
 - update of styles
 - update examples

[Screenshot]

image

image

grid-responsive

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

File: src/css/profile/mobile/common/gridview.less [hunk 1]
```diff
@@ -31,15 +29,15 @@ tau-gridview {
 		}
 
 		.ui-gridview-handler {
-			mask-image: url("images/core_grid_button_reorder.png");
+			mask-image: url("images/3_Controllers/tw_list_icon_reorder.svg");
 			mask-size: 100%;
 			background-color: @color_button_add;
```
Comment (korneliakobiela) [inline src/css/profile/mobile/common/gridview.less]: Can you change this colour value to CSS variable?

**Review by lslachciak — APPROVED** [review]
LGTM

File: src/css/profile/mobile/common/gridview.less (hunk 1 above)
Comment (TomaszLukawskiSam) [reply src/css/profile/mobile/common/gridview.less]: Done.

**Review by korneliakobiela — APPROVED** [review]
LGTM

**Review by lslachciak — APPROVED** [review]

**Pull request merged**